src/node_api.cc

```diff
@@ -888,7 +889,7 @@ napi_status napi_make_callback(napi_env env,
         reinterpret_cast<v8::Local<v8::Value>*>(const_cast<napi_value*>(argv)),
         {0, 0});
   } else {
-    auto node_async_context =
+    v8impl::AsyncContext* node_async_context =
```

Comment:
By the way, I think the results of casts and `new ...` can qualify as “obvious” types. I don’t mind changing these, but I’d avoid doing it everywhere because of the churn it creates.

Reply:
I just consider that "Forget `auto` keywords in C++ except lambda".